src/sentry/tasks/post_process.py

```diff
@@ -28,7 +28,7 @@
     from sentry.eventstore.models import Event, GroupEvent
     from sentry.eventstream.base import GroupState, GroupStates
 
-logger = logging.getLogger("sentry")
+logger = logging.getLogger(__name__)
```

Comment:
This is an important change since it allows using `labels.name=sentry.tasks.post_process` in GCP to filter out messages from this file.